File: src/lib/services/shell.ts
/**
 * Shell 服务
 * @deprecated 请使用 api.shell 代替
 */

import { api } from '$lib/api';

export const openUrl = (url: string) => api.shell.openUrl(url);
export const openPath = (path: string) => api.shell.openPath(path);
export const openApp = (app: string) => api.shell.openApp(app);

File: src-tauri/src/commands/shell.rs
//! Shell 操作命令

use crate::services::ShellService;

#[tauri::command]
pub fn open_url(url: String) -> Result<(), String> {
    ShellService::open_url(url)
}

#[tauri::command]
pub fn open_path(path: String) -> Result<(), String> {
    ShellService::open_path(path)
}

#[tauri::command]
pub fn open_app(app: String) -> Result<(), String> {
    ShellService::open_app(app)
}
